whylog/client/searchers.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, file_path):
     def search(self, search_data):
         pass
 
+    @classmethod
+    def _decrease_actual_offset_properly(cls, actual_offset, decreasion_string):
+        return actual_offset - len(decreasion_string) - 1
+
     def _reverse_from_offset(self, offset, buf_size=BufsizeConsts.STANDARD_BUF_SIZE):
```

Comment:
Any reason? At the moment, this method cannot become a classmethod, due to its dependency on its field `self._file_path`.